shop-service/product-service/handlers/getProductsById.ts

```diff
@@ -0,0 +1,23 @@
+import ProductService from "../productsService";
+import createResponse from "../helpers/createResponse";
+import { NOT_FOUND_ERROR, INTERNAL_SERVER_ERROR } from "../constants";
+
+export const getProductsById = async (event) => {
+  const { id } = event.pathParameters;
+  let response = null;
+  try {
+    const product = new ProductService().getProductById(id);
+    if (!product) {
+      response = createResponse(404, {
+        message: NOT_FOUND_ERROR,
+      });
+    } else {
+      response = createResponse(200, product);
+    }
+    return await response;
```

Comment:
"createResponse" is sync function, so it seems that we don't need to use await there.
Please consider using Promise.resolve(someData) for hardcoded mock data if you want to use await.

Reply:
Thanks! I fixed it

Reply:
I think it was a little misunderstanding :) I meant that you should use it like this (for example):
const products = await someservice.getProductsList();
return createResponse(products).

The reason is that in future you'll replace someservice (with hardcoded data) with database api, which calls are async.

Reply:
added delay response function